lib/rack/lint.rb

```diff
@@ -885,12 +950,16 @@ def initialize(stream)
           end
         end
       end
-
-      # :startdoc:
     end
   end
 end
 
 ##
 ## == Thanks
-## Some parts of this specification are adopted from {PEP 333 – Python Web Server Gateway Interface v1.0}[https://peps.python.org/pep-0333/] I'd like to thank everyone involved in that effort.
+##
+## We'd like to thank everyone who has contributed to the Rack project over the years. Your work has made this specification possible. That includes everyone who has contributed code, documentation, bug reports, and feedback. We'd also like to thank the authors of the various web servers, frameworks, and libraries that have implemented the Rack specification. Your work has helped to make the web a better place.
+##
+## Some parts of this specification are adapted from {PEP 333 – Python Web Server Gateway Interface v1.0}[https://peps.python.org/pep-0333/]. We'd like to thank everyone involved in that effort.
+##
+
+# :startdoc:
```

Comment:
```suggestion
```

AFAIK, startdoc only applies to the rest of the file, so it has no affect at the end of the file.  However, maybe there is a reason this was added?

Reply:
I agree with you, but I also found this: https://github.com/ruby/rdoc/issues/398